src/test/java/com/sap/oss/phosphor/fosstars/data/release/info/ReleaseInfoLoaderTest.java

```diff
@@ -0,0 +1,234 @@
+    releaseInfoFromMaven = spy(releaseInfoFromMaven);
+
+    CloseableHttpClient httpClient = mock(CloseableHttpClient.class);
+    when(releaseInfoFromMaven.httpClient()).thenReturn(httpClient);
+
+    CloseableHttpResponse response = mock(CloseableHttpResponse.class);
+    when(httpClient.execute(any())).thenReturn(response);
+
+    HttpEntity entity = mock(HttpEntity.class);
+    when(response.getEntity()).thenReturn(entity);
+
+    try (InputStream content = getClass().getResourceAsStream("ReleaseInfoFromMaven.json")) {
+      when(entity.getContent()).thenReturn(content);
+
+      ValueHashSet values = new ValueHashSet();
+      assertEquals(0, values.size());
+
+      ReleaseInfoLoader provider =
+          new ReleaseInfoLoader(releasesFromGitHub, releaseInfoFromMaven, releaseInfoFromNpm);
+      provider.update(MAVEN_ARTIFACT, values);
+
+      assertEquals(1, values.size());
+      assertTrue(values.has(RELEASED_ARTIFACT_VERSIONS));
+      assertTrue(values.of(RELEASED_ARTIFACT_VERSIONS).isPresent());
+      assertFalse(values.of(RELEASED_ARTIFACT_VERSIONS).get().isUnknown());
+      assertFalse(values.of(RELEASED_ARTIFACT_VERSIONS).get().get().empty());
+      assertEquals(15, values.of(RELEASED_ARTIFACT_VERSIONS).get().get().size());
+      assertTrue(values.of(RELEASED_ARTIFACT_VERSIONS).get().get().get("1.10.10").isPresent());
+    }
+  }
+
+  @Test
+  public void testIfArtifactIsNpmArtifact() throws IOException {
+    releaseInfoFromNpm = new ReleaseInfoFromNpm();
+    releaseInfoFromNpm = spy(releaseInfoFromNpm);
+
+    CloseableHttpClient httpClient = mock(CloseableHttpClient.class);
+    when(releaseInfoFromNpm.httpClient()).thenReturn(httpClient);
+
+    CloseableHttpResponse response = mock(CloseableHttpResponse.class);
+    when(httpClient.execute(any())).thenReturn(response);
+
+    HttpEntity entity = mock(HttpEntity.class);
+    when(response.getEntity()).thenReturn(entity);
+
+    try (InputStream content = getClass().getResourceAsStream("ReleaseInfoFromNpm.json")) {
+      when(entity.getContent()).thenReturn(content);
+
+      ValueHashSet values = new ValueHashSet();
+      assertEquals(0, values.size());
+
+      ReleaseInfoLoader provider =
+          new ReleaseInfoLoader(releasesFromGitHub, releaseInfoFromMaven, releaseInfoFromNpm);
+      provider.update(NPM_ARTIFACT, values);
+
+      assertEquals(1, values.size());
+      assertTrue(values.has(RELEASED_ARTIFACT_VERSIONS));
+      assertTrue(values.of(RELEASED_ARTIFACT_VERSIONS).isPresent());
+      assertFalse(values.of(RELEASED_ARTIFACT_VERSIONS).get().isUnknown());
+      assertFalse(values.of(RELEASED_ARTIFACT_VERSIONS).get().get().empty());
+      assertEquals(24, values.of(RELEASED_ARTIFACT_VERSIONS).get().get().size());
+      assertTrue(values.of(RELEASED_ARTIFACT_VERSIONS).get().get().get("0.7.1").isPresent());
+    }
+  }
+
+  @Test
+  public void testIfMavenArtifactHasNoReleaseInfoAndNoGitHubProject() throws IOException {
```

Comment:
`MAVEN_ARTIFACT` has a GitHub project. Could you please update the name of this test case?
```suggestion
  public void testIfMavenArtifactHasNoReleaseInfoOnGitHub() throws IOException {
```